cmd/check-gh-automation/main.go

```diff
@@ -190,6 +190,11 @@ func checkRepos(repos []string, bots []string, ignore sets.Set[string], client a
 
 		if configAgent != nil {
 			externalPlugins := configAgent.Config().ExternalPlugins[orgRepo]
+			repoLogger.Infof("is there a config for %s? %v", orgRepo, externalPlugins != nil)
+			if externalPlugins == nil {
+				externalPlugins = configAgent.Config().ExternalPlugins[org]
+				repoLogger.Infof("is there a config for %s? %v", org, externalPlugins != nil)
```

Comment:
remove this logging, this was only for testing

Reply:
Removed